### PR TITLE
Pin pull licenses from spdx repo

### LINKED ISSUE #3451: Pin license json
https://github.com/spdx/license-list-data/blob/38ecfa995d4d472090e6730e4edf0296c6a12ed8/json/licenses.json#L187-L198

This is a request from the store/snapd to align the license across products.

### PR DESCRIPTION
**Pull request by tbille**
## Done
- Pin spdx licenses
- Remove duplicate custom licenses that didn't exist on the previous version of the license file
- This will help aligning the licenses across store/snapd/snapcraft

## How to QA
- `dotrun clean`
- Visit some snap details pages and make sure the licenses are correct
- Visit some listing pages and make sure the licenses match the details pages and that nothign has changed

## Issue / Card
Fixes #3451

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "test-js": "jest",
     "test-js-all": "yarn run lint-js && yarn run test-js",
     "test-python": "yarn run lint-python && FLASK_DEBUG=0 python3 -m unittest discover tests",
-    "get-licenses": "if [ ! -f webapp/licenses.json ] ; then curl https://raw.githubusercontent.com/spdx/license-list-data/master/json/licenses.json -o webapp/licenses.json; fi",
+    "get-licenses": "if [ ! -f webapp/licenses.json ] ; then curl https://raw.githubusercontent.com/spdx/license-list-data/38ecfa995d4d472090e6730e4edf0296c6a12ed8/json/licenses.json -o webapp/licenses.json; fi",
```
Comment (jkfran) [inline package.json:10]: @tbille with Docker it will not be executed :|

Comment (tbille) [reply package.json]: you are right. I updated the way this list is managed, and added some documentation on how to update the list.

**Review by jkfran — APPROVED** [review]
Looks good

File: HACKING.md [hunk 2]
```diff
@@ -90,3 +90,14 @@ Now visit http://127.0.0.1:8004/_status to see the endpoints provided by Taliske
 
 - http://127.0.0.1:8004/_status/check - A basic check that the site is running
 - http://127.0.0.1:8004/_status/metrics - The prometheus metrics for the application
+
+## Update the list of licenses
+
+The licenses that we use are based on the [SPDX Specification](https://spdx.github.io/license-list-data/). In order to have all the products supporting the same set of licenses, the list needs to be syncronised between snapcraft.io, snapd and the snap store.
+
+In case you need to udpate the license list:
```
Comment (jkfran) [inline HACKING.md:98]:
```suggestion
In case you need to update the license list:
```

File: HACKING.md [hunk 3]
```diff
@@ -90,3 +90,14 @@ Now visit http://127.0.0.1:8004/_status to see the endpoints provided by Taliske
 
 - http://127.0.0.1:8004/_status/check - A basic check that the site is running
 - http://127.0.0.1:8004/_status/metrics - The prometheus metrics for the application
+
+## Update the list of licenses
+
+The licenses that we use are based on the [SPDX Specification](https://spdx.github.io/license-list-data/). In order to have all the products supporting the same set of licenses, the list needs to be syncronised between snapcraft.io, snapd and the snap store.
```
Comment (jkfran) [inline HACKING.md:96]:
```suggestion
The licenses that we use are based on the [SPDX Specification](https://spdx.github.io/license-list-data/). In order to have all the products supporting the same set of licenses, the list needs to be synchronised between snapcraft.io, snapd and the snap store.
```

**Pull request merged**